README.md

```diff
@@ -63,7 +71,7 @@ publishToStyleGuide: true
 
 Create a `Code.mdx` file alongside one of your component.
 
-Inside of `Code.mdx`. Use the `<Code />` and `<Component />` components in your MDX to fluidly author your docs and inline code snippets and rendered examples of your component.
+Inside of `Code.mdx`, simply use the `<Code />`, `<Component />`, and/or `<TableOfContents />` components in your MDX to fluidly author your docs and inline code snippets and rendered examples of your component.
```

Comment:
Let's avoid "simply" because it might make people feel dumb if they don't get it. Sorry, this is a pet peeve of mine in readmes 🙃

Reply:
totally agree, good call